tox.ini

```diff
@@ -9,97 +9,97 @@ envlist =
     py{2.7,3.4,3.5,3.6,3.7,3.8,3.9,3.10}
 
     # === Integrations ===
-    # General format is {pythonversion}-{integrationname}-{frameworkversion}
+    # General format is {pythonversion}-{integrationname}-v{frameworkversion}
```

Comment:
If we suffix the environment name with just the framework version, tox4 will try to validate the python version from it and treats the standalone version number as a python version. By adding the `v`, it prevents tox from doing this.